app/views/account/utility_providers/new.html.slim

```diff
@@ -0,0 +1,29 @@
+.container.pt-3
+  .jumbotron
+    .col-8
+      h1.text-left
+        = t('.add-utilities-provider')
+      h2.text-left
```

Comment:
This text is redandant as well
too much of useless info to read
just company info is enough